test/acceptance/tests/fixtures/cases/static-client-inject/patch.yaml

```diff
@@ -3,6 +3,7 @@ kind: Deployment
 metadata:
   name: static-client
 spec:
+
```

Comment:
?

Reply:
Just checking if you're paying attention 😄